pom.xml

```diff
@@ -179,6 +179,7 @@
                             <exclude>**/rocksdb-*/**</exclude>
                             <exclude>**/hbase-*/**</exclude>
                             <exclude>**/apache-cassandra-*/**</exclude>
+                            <exclude>**/pid</exclude>
```

Comment:
maybe should exclude the whole generated dir? (ignore the suggestion if it's just a test)

Reply:
These files or directories are excluded because validation is performed before using `mvn test`, which may include data generated from previous tests or files generated during server runtime, such as:

```text
/home/runner/work/incubator-hugegraph/incubator-hugegraph/hugegraph-server/apache-hugegraph-incubating-1.2.0/bin/pid
/home/runner/work/incubator-hugegraph/incubator-hugegraph/hugegraph-server/hugegraph-test/rocksdb-data/graph/m/OPTIONS-
```

Reply:
fine